### PR TITLE
Instruct users to clone specific branch of samples instead of master 

### PR DESCRIPTION
**Pull request by HansK-p**
Instruct users to clone specific branch of samples instead of master 

# Description

Replaced 'git clone https://github.com/dapr/samples.git' with 'git cl…one -b <dapr_version_tag> https://github.com/dapr/samples.git'. The change was made for all relevant samples except the "Dapr with Docker-Compose" sample as that one contains a Docker Compose file which refers to "daprio/daprd:edge". The edge image should always be compatible with the Daper samples master branch.

## Issue reference

dapriosamples/hello-k8s-node:edge as used in sample "2.hello-kubernetes" fails with Dapr 0.8.0 as DAPR_HTTP_PORT and DAPR_GRPC_PORT isn't set #251

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The sample code compiles correctly
* [x] You've tested new builds of the sample if you changed sample code
* [x] You've updated the sample's README if necessary

### CODE REVIEW

File: 1.hello-world/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Follow [instructions](https://github.com/dapr/docs/blob/master/getting-started/e
 Now that we've locally set up Dapr, clone the repo, then navigate to the Hello World sample: 
 
 ```bash
-git clone https://github.com/dapr/samples.git
```
Comment (pruthvidhodda) [inline 1.hello-world/README.md:27]: It'd be nice if we can point to the supported versions table on landing page. We should also take into account people using `edge` version of dapr runtime. So, can we please add the instructions like below:

```bash 
git clone -b [<dapr_version_tag>](https://github.com/dapr/samples#supported-dapr-runtime-version) https://github.com/dapr/samples.git
```
> Note: Ignore checking out the tag if using `edge` version of dapr runtime. Use `git clone https://github.com/dapr/samples.git` instead.

```bash
cd samples/1.hello-world
```

Comment (pruthvidhodda) [reply 1.hello-world/README.md:27]: Thanks for updating. I have merged master into this branch and there is a new sample in this branch now. So please do the same change for 11.hello-dapr-slim as well.

Comment (HansK-p) [reply 1.hello-world/README.md:27]: This is hopefully ok now.

Br

Hans K.

**Comment by pruthvidhodda** [comment]
@HansK-p thanks for your contribution :)

**Comment by HansK-p** [comment]
@pruthvidhodda No problem:). This is one of the strengths of open source, and Dapr is an interesting open source project....

**Pull request merged**